content/posts/2023/reana-0.9.0.md

```diff
@@ -0,0 +1,161 @@
+to learn more about this feature.
+
+### Notifications to inform when critical levels of quota usage is reached
+
+A new notifications feature has been added to REANA web interface to alert users
+when their [quota](https://docs.reana.io/advanced-usage/user-quotas) is
+approaching or has reached its limit. This feature is intended to help users
+stay within their allocated Disk space and CPU consumption limits and inform
+them when deletion of unnecessary workflow runs should be performed in order to
+liberate some resources.
+
+![Quota notifications](/images/reana-0.9.0-notifications.png)
+
+## What's new for the administrators?
+
+### Configure global workspace retention rules
+
+Workspace retention rules are giving users the possibility to remove unwanted files from their workflows' workspaces.
+However, administrators can also define a global workspace file retention period, after which all the files that are not listed as inputs or outputs of a workflow will be deleted, regardless of user-defined workspace retention rules.
+This can be achieved by setting the `workspaces.retention_rules.maximum_period` Helm value to the desired amount of retention days.
+Please note that, by default, files are retained forever.
+
+For more information about this, see the related documentation page about [configuring global workspace retention rules](https://docs.reana.io/administration/configuration/configuring-global-workspace-retention-rules).
+
+### Configure an additional volume for infrastructure components
+
+Up until now, REANA has used only one shared storage volume to store all the workflows' workspaces, together with the database's and message broker's data.
+This means that, if the shared storage became full, some of the REANA components would fail due to the lack of free disk space.
+While this is still the default behaviour of REANA, now it is also possible to configure a second infrastructure storage volume that can be used to separate workflows' workspaces from the data stored by REANA's infrastructure components.
+
+If you are interested in this change, please see the documentation page about [configuring storage volumes](https://docs.reana.io/administration/configuration/configuring-storage-volumes).
+
+### Configure scheduler requeue count for busy cluster situations
+
+Under specific circumstances, for example when the REANA cluster is busy, it can happen that some workflows cannot be scheduled.
+These workflows are then queued again so that REANA can retry scheduling them in the future.
+However, workflows with high priority that are requeued many times can overwhelm the queue, preventing other workflows from being executed.
+
+To avoid this, REANA 0.9.0 introduces a limit on the number of scheduling retries of each workflow. This limit is set to 200 retries by default, but it can be customized with the `reana_server.environment.REANA_SCHEDULER_REQUEUE_COUNT` Helm value.
```

Comment:
The formatting of the phrase looks funny on my machine:

![200-retries](https://user-images.githubusercontent.com/517546/214850136-55d6d3a5-1428-4d6e-8553-1c190735ddb2.png)

Perhaps we can reformulate to "cut the phrase" at better places?  Or expand to say also about `REANA_SCHEDULER_REQUEUE_SLEEP` that people should configure together.

For example: This limit is set to 200 retries by default, after which is incoming workflow will be declared as impossible to schedule. The value of 200 retries can be customised with the .... Helm value. You may also want to customise the accompanying value REANA_SCHEDULER_REQUEUE_SLEEP defining time interval between retries.

(did not try how it looks, just jotted down some random phrases to try improving the line break)

Reply:
BTW general remark: shall we show YAML snippets to demonstrate configuration examples for each advertised feature? 

OT1H, it might make the blog more "alive", as it were.
OTOH, we are more interested in explaining the user-facing features, not admin-facing features, so this might do.